packages/client/src/graphql/mutations/login.gql

```diff
@@ -0,0 +1,32 @@
+mutation Login($email: String!, $password: String!) {
+  login(input: { email: $email, password: $password }) {
+    __typename
+
+    ... on User {
+      id
+      name
+      phone
+      email
+    }
+
+    ... on FieldsValidationError {
+      message
+      fieldErrors {
+        path
+        message
+      }
+    }
+
+    ... on BadCredentialsError {
+      message
+    }
+
+    ... on NonExistingUserError {
+      message
+    }
```

Comment:
we can remove these ones, since all errors with only a message field are included in the `IError` block.